pkg/server/options/flags.go

```diff
@@ -176,6 +176,9 @@ var (
 
 		// KCP Virtual Workspaces flags
 		"virtual-workspace-address", // Address of a stand-alone virtual workspace apiserver.
+
+		// Kubernetes ServiceAccount Controller
+		"root-ca-file", // If set, this root certificate authority will be included in service account's token secret. This must be a valid PEM-encoded CA bundle.
```

Comment:
is this required after the PR ?

Reply:
dropoed, thanks!

Reply:
turns out, that CI complains about it...  do you want me to disallow that flag?

Reply:
Suggest rebasing on #681

Reply:
rebased.